azure-stack/hci/concepts/system-requirements.md

```diff
@@ -72,6 +72,9 @@ Here are the Azure requirements for your Azure Stack HCI cluster:
 
 A standard Azure Stack HCI cluster requires a minimum of one server and a maximum of 16 servers.
 
+> [!IMPORTANT]
```

Comment:
This would be better as a bullet rather than an Important reader element. I'll make a recommendation below.